mesh_sequence_controller.py

```diff
@@ -37,6 +37,25 @@
 import glob
 from bpy.app.handlers import persistent
 
+import re
+
+def tryint(s):
+    try:
+        return int(s)
+    except:
+        return s
+
+def alphanum_key(s):

```

Comment:
Change `s` to something more descriptive. Also let's do `alphanumKey`

Reply:
I removed the `tryint()` function for performance. And `alphanumKey` was changed.